tests/integration/cass_integration_test.go

```diff
@@ -0,0 +1,168 @@
+package integration
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/k8ssandra/k8ssandra/tests/integration/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	chartName  = "cass-operator"
+	chartPath  = "../../charts/k8ssandra/charts/cass-operator"
+	configPath = "C:\\Users\\Jeff Banks\\.kube\\config"
+
+	crdClusterYaml = "../../charts/k8ssandra-cluster/Chart.yaml"
+
+	defaultNamespace        = "default"
+	deleteCRDTimeoutSeconds = 15
+
+	k8ssandraNamespace  = "k8ssandra-ns"
+	operatorReleaseName = "k8ssandra-1"
+	clusterReleaseName  = "cluster-1"
+	clusterName         = "dc1"
+)
+
+// TestCassOperator performs basic installation of k8ssandra operator and c* cluster using setup & teardown.
+func TestCassOperatorAndClusterInstall(t *testing.T) {
```

Comment:
Does this test verify that the CassandraDatacenter is fully initialized, i.e., ready? More than anything else, that is what I want to see in terms of verification.

Reply:
The initial set of logic is for the setup and verification is primarily focused on the cass-operator.  There is an issue getting past that step related to Prometheus error being reported upon install.  

Adding to a summary comment for this PR (coming soon), with a deconstructed view of what is being verified.  That is, separating into two units of work for verification of:
- k8ssandra (cass-operator) referenced
- k8ssandra (k8ssandra-cluster)